### PR TITLE
Update virtual_authenticator.zh-cn.md

### PR DESCRIPTION
**Pull request by iambocai**
Add  chinese translation of page <Virtual Authenticator>

**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to review and merge it quickly**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, and help reviewers by making them as simple and short as possible.

### Types of changes

- [ ] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [ ] Code example added (and I also added the example to all translated languages)
- [x] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

File: website_and_docs/content/documentation/webdriver/interactions/virtual_authenticator.zh-cn.md [hunk 1]
```diff
@@ -89,9 +80,9 @@ Removes the previously added virtual authenticator.
 {{< /tab >}}
 {{< /tabpane >}}
 
-## Create Resident Credential
+## 创建永久凭据
 
-Creates a resident (stateful) credential with the given required credential [parameters](https://w3c.github.io/webauthn/#sctn-automation-add-credential). 
+使用给定的所需凭据 [参数] (https://w3c.github.io/webauthn/#sctn-automation-add-credential) 创建一个永久凭据。
```
Comment (alaahong) [inline website_and_docs/content/documentation/webdriver/interactions/virtual_authenticator.zh-cn.md]: Additional space?
image

So, as you selected “I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.” Had you really compiled locally？

Comment (alaahong) [reply website_and_docs/content/documentation/webdriver/interactions/virtual_authenticator.zh-cn.md]: Seems missed the original highlight word

File: website_and_docs/content/documentation/webdriver/interactions/virtual_authenticator.zh-cn.md [hunk 2]
```diff
@@ -110,9 +101,9 @@ Creates a resident (stateful) credential with the given required credential [par
 {{< /tab >}}
 {{< /tabpane >}}
 
-## Create Non-Resident Credential 
+## 创建临时凭据
 
-Creates a resident (stateless) credential with the given required credential [parameters](https://w3c.github.io/webauthn/#sctn-automation-add-credential). 
+使用给定的所需凭据 [参数] (https://w3c.github.io/webauthn/#sctn-automation-add-credential) 创建一个临时凭据。
```
Comment (alaahong) [inline website_and_docs/content/documentation/webdriver/interactions/virtual_authenticator.zh-cn.md]: How about the status?

**Review by alaahong — DISMISSED** [review]
Can you take a look on the comment part?

File: website_and_docs/content/documentation/webdriver/interactions/virtual_authenticator.zh-cn.md [hunk 3]
```diff
@@ -236,4 +227,4 @@ Sets whether the authenticator will simulate success or fail on user verificatio
 {{< /tab >}}
 {{< tab header="Kotlin" >}}
 {{< /tab >}}
-{{< /tabpane >}}
\ No newline at end of file
+{{< /tabpane >}}
```
Comment (alaahong) [inline website_and_docs/content/documentation/webdriver/interactions/virtual_authenticator.zh-cn.md:230]: Additional change?

**Comment by diemol** [comment]
@alaahong do you think you can make the requested changes so we merge these changes?

**Comment by alaahong** [comment]
> @alaahong do you think you can make the requested changes so we merge these changes?

Sure, just commit. A bit confused on the resident credential, just treat as fixed/persisted credential.

**Review by diemol — APPROVED** [review]
Thank you, @alaahong!

**Pull request merged**